pyproject.toml

```diff
@@ -68,7 +68,7 @@ smart-open = "^1.5.7"
 structlog = "^20.1.0"
 structlog-sentry = "^1.2.2"
 urllib3 = "^1.24.2"
-uwsgi = "2.0.22"
+uwsgi = "2.0.25"
```

Comment:
Upgrading `pylint` below to the latest version ("^3.2.0") gets rid of the astroid exceptions when running pylint, but there are still some new pylint errors (`use-dict-literal` and `missing-timeout`) scattered throughout the code base.

Reply:
Yeah, I'm thinking I'll probably use this opportunity to cut over to Ruff